bitcoin/src/address/mod.rs

```diff
@@ -475,8 +475,14 @@ impl<V: NetworkValidation> Address<V> {
     }
 
     /// Marks the network of this address as unchecked.
+    pub fn to_unchecked(self) -> Address<NetworkUnchecked> {
+        Address::from_inner(self.to_inner())
+    }
+
+    /// Marks the network of this address as unchecked.
+    #[deprecated(since = "0.33.0", note = "use to_unchecked instead")]
     pub fn into_unchecked(self) -> Address<NetworkUnchecked> {
-        Address::from_inner(self.into_inner())
+        Address::from_inner(self.to_inner())
```

Comment:
For next time; typically when we deprecate a function we just call through to the new one in the body of the old one. Especially since you had to change the body anyways. But for now this is good.